taf/updater/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class UpdateFailed(Exception):
```

Comment:
It is a convention to add `Error` suffix if exception is actually an error.

Reply:
Also, there is already `taf.exceptions` module, can you move this exception to it? Can you add `Error` suffix to all of them, as well?
It is a good practice to have base exception like `TAFError` and to subclass it for all specific TAF errors.
Like here: https://github.com/openlawlibrary/taf-sc/blob/danixeee/api-tests/taf_sc/exceptions.py#L5

Reply:
Yeah, I've done so in the past, but it slipped my mind. Will update.

Reply:
Done